addons/phantom_camera/scripts/phantom_camera/phantom_camera_properties.gd

```diff
@@ -314,6 +321,9 @@ func set_follow_properties(property: StringName, value, pcam: Node):
 #			_:
 #				set_process(pcam, true)
 
+	if property == Constants.FOLLOW_PIXEL_PERFECT_PROPERTY_NAME:
+		follow_pixel_perfect = value
```

Comment:
Same with this, would also be nice to have it in the same area as the `_get_property_list()` above. I.e. below the `Properties.set_follow_properties(property, value, self)` function on line 178 in `PhantomCamera2D.gd`.